userspace/libscap/CMakeLists.txt

```
@@ -136,9 +136,8 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	include(FindMakedev)
 endif()
 
-
-# gVisor is currently only supported on Linux
-if(NOT MINIMAL_BUILD AND NOT WIN32 AND NOT APPLE)
+# gVisor is currently only supported on Linux x86_64
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64" AND NOT MINIMAL_BUILD AND NOT WIN32 AND NOT APPLE)
```

```suggestion
if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64" AND CMAKE_SYSTEM_NAME MATCHES "Linux" AND NOT MINIMAL_BUILD)
```

Since `NOT WIN32 AND NOT APPLE` can match Unix and others.

---

Updated as suggested, plus removed `${}` from both as it looks like they are not necessary